core/javatests/androidx/test/core/app/testing/BUILD.bazel

```diff
@@ -14,6 +14,8 @@ android_library(
     visibility = ["//visibility:public"],
     deps = [
         "//:androidx_annotation",
+        "@maven//:com_google_guava_guava",
+        "@maven//:junit_junit",
```

Comment:
@brettchabot isn't this a huge change? Adding 2 extra dependencies will bloat users.

JUnit shouldn't be a dependency since people might choose different testing library, e.g. JUnit 5. Also just for assertFalse it seems silly since there's `assertThat().isFalse` already there?

Guava is huge in general and this module was so lightweight before. Also is it getting repackaged automatically?